srcpkgs/bitwig-studio/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'bitwig-studio'
+
+pkgname=bitwig-studio
+homepage="bitwig.com"
+version=3.2.6
+revision=1
+archs="x86_64"
+short_desc="DAW for music production, remixing and live performance"
+
```

Comment:
Hi, could you remove these blank lines between variables? Then xlint will be able to tell you how to reorder lines.